### DISCUSSION TITLE
dashy update process

### DISCUSSION THREAD

**Discussion by dougharvey51** [tteck/Proxmox#628] [General]
The docs for the dashy script say that after you modify the config you need to:
"go into update configuration and rebuild application."
I could not find what this meant, but found that if I use the proxmox console for the dashy container and:

cd /dashy/public
yarn build

... that all my changes persist between container restarts.

**Comment by l3gitpanda** [comment]
So basically whenever you make changes you need to save it to disk and then go to the wrench in the top right and then press on rebuild application. You save it to disk by pressing on the pencil in the top right and then pressing on save to disk.
